client/src/actions/Cart.actions.tsx

```diff
@@ -0,0 +1,16 @@
+import { CartItem } from "./../interfaces";
+import { PRICE_FRACTION_DIGITS } from "../constants";
+
+export const getTotalPrice = (cartItems: CartItem[]) => {
+  let tot_price = 0.0;
```

Comment:
```suggestion
  const tot_price = 0.0;
```

Reply:
the value of `tot_price` will be updated in the for loop though?